app/build.gradle

```diff
@@ -50,33 +50,52 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
     kotlinOptions {
+        freeCompilerArgs=['-Xjvm-default=all']
         jvmTarget = '1.8'
     }
     buildFeatures {
         viewBinding true
     }
+    splits {
+        abi {
+            enable true
+            reset()
+            include "armeabi-v7a", "arm64-v8a"
+            universalApk true
+        }
+    }
 }
 
 dependencies {
+    def arklibVersion = '0.1.0-SNAPSHOT-f81e34a657'
+    releaseImplementation "space.taran:arklib:$arklibVersion"
+//    debugImplementation "space.taran:arklib-debug:$arklibVersion"
```

Comment:
//

Reply:
Actually debug version is very slow and doesn't seem to be needed